apps/mobile/src/libs/components/link/link.tsx

```diff
@@ -1,16 +1,19 @@
 import { Link as UILink } from "@react-navigation/native";
 import React, { type ComponentProps } from "react";
 
+import { type StyleProp, type TextStyle } from "~/libs/types/types";
+
 import { styles } from "./styles";
 
 type Properties = {
 	label: string;
+	style?: StyleProp<TextStyle>;
 	to: ComponentProps<typeof UILink>["to"];
 };
 
-const Link: React.FC<Properties> = ({ label, to }) => {
+const Link: React.FC<Properties> = ({ label, style, to }) => {
 	return (
-		<UILink style={styles.link} to={to}>
+		<UILink style={[styles.link, style]} to={to}>
 			{label}
```

Comment:
Can we use some of our global titles (presetToStyleMap) or global fonts (sizeToStyleMap)?

Reply:
I can move `fontWeightToFamilyMap`, `presetToStyleMap`, `sizeToStyleMap`, to `~/libs/maps/maps` and refactor it to be set like our Text component.